src/test/java/hw6/SomeTest.java

```diff
@@ -0,0 +1,66 @@
+package hw6;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import hw6.domain.MetalAndColor;
+import hw6.domain.User;
+import hw6.pages.HomePage;
+import org.testng.annotations.*;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Iterator;
+import java.util.List;
+
+import static hw6.JdiSite.metalsAndColorsPage;
+
+
+public class SomeTest {
+
+    @BeforeSuite
+    public void beforeClass() {
+        PageFactory.initElements(JdiSite.class);
+    }
+
+    @BeforeClass
+    public void testLogin() {
```

Comment:
This is not the test

Reply:
The name have been updated to  `loginUser`.